Tools/_sunrise/test_pr_check_summary.py

```
@@ -0,0 +1,174 @@
+        jobs = [
+            {
+                "name": "build",
+                "status": "completed",
+                "conclusion": "failure",
+                "html_url": "https://example.test/build",
+                "id": 1,
+            }
+        ]
+
+        class FakeClient:
+            def download_job_log(self, job_id):
```

Ruff (0.15.11)

[warning] Missing return type annotation for private function `download_job_log`

Add return type annotation: `str`

(ANN202)